data/argentium-snek.json

```diff
@@ -1,8 +1,18 @@
 {
-	"name": "Argentium Snek",
+	"names": [
+		"Argentium Snek"
+	],
+	"description": "A snek representing the argentium award on Reddit",
 	"pixelated": true,
 	"firstAppearance": {
 		"date": "2021-03-23T00:32:37.760Z",
 		"source": "https://discord.com/channels/429823323585773568/430527417791152140/823715829950054430"
-	}
+	},
+	"artists": [
+		{
+			"name": "haykam",
+			"discord": 412782876304867329,
```

Comment:
Due to numerical precision restrictions, snowflakes should be represented as strings.